modules/relevadRtdProvider.js

```diff
@@ -0,0 +1,367 @@
+            serverData.rawdata = data;
+            if (data) {
+              addRtdData(reqBidsConfigObj, data, moduleConfig);
+            }
+          } catch (e) {
+            logError(SUBMODULE_NAME, 'unable to parse data: ' + e);
+          }
+          onDone();
+        }
+      },
+      error: function () {
+        logError(SUBMODULE_NAME, 'unable to receive data');
+        onDone();
+      }
+    },
+    null,
+    { method: 'GET', ...AJAX_OPTIONS, },
+  );
+}
+
+/**
+ * Sets global ORTB user and site data
+ *
+ * @param      {dictionary}  ortb2     The gloabl ORTB structure
+ * @param      {dictionary}  rtdData   Rtd segments and categories
+ */
+export function setGlobalOrtb2(ortb2, rtdData) {
+  try {
+    let addOrtb2 = composeOrtb2Data(rtdData, 'site');
+    !isEmpty(addOrtb2) && mergeDeep(ortb2, addOrtb2);
+  } catch (e) {
+    logError(e)
+  }
+}
+
+/**
+ * Compose ORTB2 data fragment from RTD data
+ *
+ * @param  {dictionary}  rtdData RTD segments and categories
+ * @param  {string}      prefix  Site path prefix
+ * @return {dictionary} ORTB2 fragment ready to be merged into global or bidder ORTB
+ */
+function composeOrtb2Data(rtdData, prefix) {
+  const segments = rtdData.segments;
+  const categories = rtdData.categories;
+  const content = rtdData.content;
+  let addOrtb2 = {};
+
+  !isEmpty(segments) && deepSetValue(addOrtb2, 'user.ext.data.relevad_rtd', segments);
+  !isEmpty(categories.cat) && deepSetValue(addOrtb2, prefix + '.cat', categories.cat);
+  !isEmpty(categories.pagecat) && deepSetValue(addOrtb2, prefix + '.pagecat', categories.pagecat);
+  !isEmpty(categories.sectioncat) && deepSetValue(addOrtb2, prefix + '.sectioncat', categories.sectioncat);
+  categories.cattax && deepSetValue(addOrtb2, prefix + '.cattax', categories.cattax);
+
+  if (!isEmpty(content) && !isEmpty(content.segs) && content.segtax) {
+    const contentSegments = {
+      name: 'relevad',
+      ext: { segtax: content.segtax },
+      segment: content.segs.map(x => { return {id: x}; })
+    };
+    deepSetValue(addOrtb2, prefix + '.content.data', [contentSegments]);
+  }
+  return addOrtb2;
+}
+
+/**
+ * Sets ORTB user and site data for a given bidder
+ *
+ * @param      {dictionary}  bidderOrtbFragment  The bidder ORTB fragment
+ * @param      {object}  bidder     The bidder name
+ * @param      {object}  rtdData    RTD categories and segments
+ */
+function setBidderSiteAndContent(bidderOrtbFragment, bidder, rtdData) {
+  try {
+    let addOrtb2 = composeOrtb2Data(rtdData, 'site');
+    !isEmpty(rtdData.segments) && deepSetValue(addOrtb2, 'user.ext.data.relevad_rtd', rtdData.segments);
+    !isEmpty(rtdData.categories?.sectioncat) && deepSetValue(addOrtb2, 'site.ext.data.relevad_rtd', rtdData.categories.sectioncat);
+    if (isEmpty(addOrtb2)) {
+      return;
+    }
+    bidderOrtbFragment[bidder] = bidderOrtbFragment[bidder] || {};
+    mergeDeep(bidderOrtbFragment[bidder], addOrtb2);
+  } catch (e) {
+    logError(e)
```

Comment:
include submodule name prefix here and other areas of the PR where it's missing, thank you